src/components/10-atoms/fieldset/README.md

```diff
@@ -74,6 +74,10 @@ The `horizontal` attribute is typically used to ensure correct layout for a grou
 
 `horizontal` mode is responsive: when crossing the `sm(all)` breakpoint (575px), this component forces conformant children to assume 100% width and rearranges them into a vertical layout with correct vertical spacing.
 
+### enableresponsivestretch
+
+Boolean `enableresponsivestretch` prevents the radio buttons to use the full length in a horizontal fieldset. Instead they are aligned side by side. It must be combined with `button`and `nogap` attributes from the axa-radio.
```

Comment:
This description sentence nicely illustrates that `enableresponsivestretch` is actually a submode of `horizontal`! This in turn means that `horizontal` should not be a Boolean attribute but a string-valued one. Inspection of the present CSS for axa-fieldset shows that no selector refers to the actual value of `horizontal`, i.e. such a change is entirely compatible with the existing CSS.

I suggest to rework this PR to capture the dependency of the new behaviour on `horizontal`!

Reply:
A possible name for the value of `horizontal` is `responsivestretch` or just `stretch` (I prefer the latter one).

Reply:
Done